### ISSUE TITLE
Can't reproduce the results according to the latest mmcv3

### ISSUE THREAD

**Issue by ithmz** [hirotomusiker/CLRerNet#87]
as title

**Comment by hirotomusiker** [comment]
Thanks, please share your evaluation results if possible.

**Comment by ithmz** [comment]
Perhaps I got error at this line https://github.com/hirotomusiker/CLRerNet/blob/dae038f67da57e292e5293a68c9c1c2922de13c2/libs/models/dense_heads/clrernet_head.py#L353 to 355. so I ask AI to fix in order to run the training.
Could it be the problem? Could you run training with successful using the same environment?

**Comment by hirotomusiker** [comment]
What does the error message say?
Probably the issue comes from segmentation mask labels. 

Is segmentation mask correctly loaded here? (mask is not None?)
https://github.com/hirotomusiker/CLRerNet/blob/main/libs/datasets/culane_dataset.py#L158

**Comment by ithmz** [comment]
Yes, I'm not at my desk right now, but the error is related to `NoneType`,
I didn't modify anything so presumably I would run it without error?
I already follow the Dataset preparation step

**Comment by hirotomusiker** [comment]
We have prepared the complete dataset preparation docs:
https://github.com/hirotomusiker/CLRerNet/pull/88
Please let me know in case it doesn't work.
Thank you.

**Comment by ithmz** [comment]
The training is working fine now without a problem, but the issue is the model checkpoint size is very large, do we need to do any postprocessing step to reduce the model size

**Comment by hirotomusiker** [comment]
Could you share the config you are using and the checkpoint size?

**Comment by ithmz** [comment]
I use this config: https://github.com/hirotomusiker/CLRerNet/blob/main/configs/clrernet/culane/clrernet_culane_dla34_ema.py
and the checkpoint size is roughly 260MB

**Comment by hirotomusiker** [comment]
A checkpoint has the following components:
- meta, message_hub: training information (30kb total)
- state_dict: main training weights (61MB)  <<< this is what you need after training
- ema_state_dict (61MB): Exponential Moving Average (EMA) weights  <<< you need this when you resume training
- optimizer (121MB): optimizer parameters   <<< you need this when you resume training


After training: you can load the checkpoint with torch.load() and discard `meta`, `message_hub`, `ema_state_dict` and `optimizer, and  save a new file

During training: you can change the [checkpointing behavior](https://mmengine.readthedocs.io/en/latest/api/generated/mmengine.hooks.CheckpointHook.html) with adding parameters in [here](https://github.com/hirotomusiker/CLRerNet/blob/dae038f67da57e292e5293a68c9c1c2922de13c2/configs/_base_/default_runtime.py#L9).

**Comment by ithmz** [comment]
I succesfully trained and strip the checkpoint, but there is an issue here. I ran my trained model and the pretrained model here, the result is quite different (given that I use the same config, threshold, etc.)

pretrained model
<img width="1301" height="733" alt="Image" src="https://github.com/user-attachments/assets/80bbf19f-a210-4c91-9c56-6ac071aed3d0" />

my trained model

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/135cd91c-8917-4ce9-ba15-eb741cc2f759" />

**Comment by hirotomusiker** [comment]
Thank you,
please make sure that your model shows the similar F1_0.5 score on CULane test (https://github.com/hirotomusiker/CLRerNet?tab=readme-ov-file#test), and the inference code is exactly the same (your 2nd image shows dashed lines for prediction, different from our visualization).

**Comment by ithmz** [comment]
Fixed, I was running inference on ONNX and modify some ONNX ops that why the accuracy drops!
Thank you!
